packages/gestalt/src/icons/file-unknown.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="m11.754 16.25c-.483 0-.875.392-.8740019.876 0 .483.3920019.874.8750019.874s.875-.392.875-.875-.392-.875-.875-.875zm0-7.88c-1.519 0-2.754 1.208-2.754 2.69 0 .416.338.754.754.754s.754-.338.754-.754c0-.65.56-1.18 1.246-1.18s1.246.53 1.246 1.18c0 .397-.23.802-.559.985-.835.465-1.44 1.55-1.44 2.58-.015.416.31.766.726.781s.766-.31.781-.726c.001-.018.001-.037 0-.055 0-.524.351-1.086.665-1.261.811-.45 1.335-1.355 1.335-2.304 0-1.482-1.235-2.688-2.754-2.688zm8.66-1.956-4.828-4.828c-.375-.375-.884-.586-1.414-.586h-8.672c-1.381 0-2.5 1.119-2.5 2.5v17c0 1.381 1.119 2.5 2.5 2.5h13c1.381 0 2.5-1.119 2.5-2.5v-12.672c0-.53-.211-1.039-.586-1.414zm-1.914 14.086h-13v-17h8.5v3c0 .828.672 1.5 1.5 1.5h3z" fill="#8e8e8e" fill-rule="evenodd"/></svg>
```

Comment:
Can you check again with the PDS team? This SVG is not optimized, a simple pass through SVGO yields a 26% reduction in size
![image](https://user-images.githubusercontent.com/7877010/62163856-1c584780-b2d0-11e9-8e0c-7e68b1c6c1e2.png)